README.md

```diff
@@ -14,12 +14,14 @@ products:
 
 # Azure IoT Edge Watchdog
 
-This repo introduces the Azure IoT Edge Watchdog pattern which is split into (1) edge and (2) cloud code joined together
-by a (3) common message structure.  This pattern demonstrates a roundtrip verification of a heartbeat message sent from
-an IoT Edge device to an Azure cloud function.  If the edge device does not receive an acknowledgement of its message,
-or if the message is out of the accepted time boundary, the watchdog code will, by default, expand the window of time 
-in which it accepts an acknowledgement.  Within the Edge code, there is a stubbed method for what additional behavior 
-should happen if the Edge device does not receive an acknowledgement.
+This repo introduces the Azure IoT Edge Watchdog pattern which is split into (1) edge and (2)
+cloud code joined together by a (3) common message structure.  This pattern demonstrates a
+roundtrip verification of a heartbeat message sent from an IoT Edge device to an Azure cloud
+function.  If the edge device does not receive an acknowledgement of its message, or if the
+message is out of the accepted time boundary, the watchdog code will, by default, expand the
+window of time in which it accepts an acknowledgement.  Within the Edge code, there is a stubbed
+method for what additional behavior should happen if the Edge device does not receive an
+acknowledgement.
 

```

Comment:
Should we add a section to list out the required azure resources you need? e.g. EH Namespace with 2 EHs, and IoT Hub, IoT Edge deployment (link to quickstart vm), etc.?